test_insufficient_gas_reverts fails, locally & remotely

### Describe the bug

Run the following, and you'll see it fail.
```console
brownie test util/test/test_DFRewards_allocate_gas.py::test_insufficient_gas_reverts -s
```

Tracebacks below.

It also fails remotely. [Example log](https://github.com/oceanprotocol/df-py/actions/runs/4944277372/jobs/8839584379?pr=567)

### Traceback - Local

```text
(venv) trentmc@trentmc-xps:~/code/df-py$ brownie test util/test/test_DFRewards_allocate_gas.py::test_insufficient_gas_reverts -s
Brownie v1.19.3 - Python development framework for Ethereum

======================================================================================== test session starts ========================================================================================
platform linux -- Python 3.10.6, pytest-6.2.5, py-1.11.0, pluggy-1.0.0 -- /home/trentmc/code/df-py/venv/bin/python3
cachedir: .pytest_cache
hypothesis profile 'brownie-verbose' -> verbosity=2, deadline=None, max_examples=50, stateful_step_count=10, report_multiple_bugs=False, database=DirectoryBasedExampleDatabase(PosixPath('/home/trentmc/.brownie/hypothesis'))
rootdir: /home/trentmc/code/df-py, configfile: pytest.ini
plugins: eth-brownie-1.19.3, web3-5.31.3, hypothesis-6.27.3, forked-1.4.0, xdist-1.34.0
collected 1 item                                                                                                                                                                                    

util/test/test_DFRewards_allocate_gas.py::test_insufficient_gas_reverts RUNNING
fillAccountsWithToken(OCEAN), balances after:
  Account #0 has 8981000.0 OCEAN
  Account #1 has 1000.0 OCEAN
  Account #2 has 1000.0 OCEAN
  Account #3 has 1000.0 OCEAN
  Account #4 has 1000.0 OCEAN
  Account #5 has 1000.0 OCEAN
  Account #6 has 1000.0 OCEAN
  Account #7 has 1000.0 OCEAN
  Account #8 has 1000.0 OCEAN
  Account #9 has 1000.0 OCEAN
util/test/test_DFRewards_allocate_gas.py::test_insufficient_gas_reverts FAILED

============================================================================================= FAILURES ==============================================================================================
___________________________________________________________________________________ test_insufficient_gas_reverts ___________________________________________________________________________________

    @enforce_types
    def test_insufficient_gas_reverts():
        account0 = brownie.network.accounts[0]
        addresses, rewards, token_addr, df_rewards = _prep_batch_allocate(1250)
        with pytest.raises(Exception) as e_info:
            df_rewards.allocate(
                addresses, rewards, token_addr, {"from": account0, "gas_limit": 100000}
            )
>       assert str(e_info.value) == "base fee exceeds gas limit"
E       AssertionError: assert 'Execution re...on parameter.' == 'base fee exceeds gas limit'
E         - base fee exceeds gas limit
E         + Execution reverted during call: 'VM Exception while processing transaction: out of gas'. This transaction will likely revert. If you wish to broadcast, include `allow_revert:True` as a transaction parameter.

util/test/test_DFRewards_allocate_gas.py:48: AssertionError
util/test/test_DFRewards_allocate_gas.py::test_insufficient_gas_reverts ====================================================================================== short test summary info ======================================================================================
FAILED util/test/test_DFRewards_allocate_gas.py::test_insufficient_gas_reverts - AssertionError: assert 'Execution re...on parameter.' == 'base fee exceeds gas limit'
======================================================================================== 1 failed in 12.87s =================
```

### Traceback - Remote

```text
________________________ test_insufficient_gas_reverts _________________________

    @enforce_types
    def test_insufficient_gas_reverts():
        account0 = brownie.network.accounts[0]
        addresses, rewards, token_addr, df_rewards = _prep_batch_allocate(1250)
        with pytest.raises(Exception) as e_info:
            df_rewards.allocate(
                addresses, rewards, token_addr, {"from": account0, "gas_limit": 100000}
            )
>       assert str(e_info.value) == "base fee exceeds gas limit"
E       AssertionError: assert 'intrinsic gas too low' == 'base fee exceeds gas limit'
E         - base fee exceeds gas limit
E         + intrinsic gas too low

util/test/test_DFRewards_allocate_gas.py:48: AssertionError
---------------------------- Captured stdout setup -----------------------------
fillAccountsWithToken(OCEAN), balances after:
  Account #0 has 991000.0 OCEAN
  Account #1 has 1000.0 OCEAN
  Account #2 has 1000.0 OCEAN
  Account #3 has 1000.0 OCEAN
  Account #4 has 1000.0 OCEAN
  Account #5 has 1000.0 OCEAN
  Account #6 has 1000.0 OCEAN
  Account #7 has 1000.0 OCEAN
  Account #8 has 1000.0 OCEAN
  Account #9 has 1000.0 OCEAN
```

--- a/util/test/test_DFRewards_allocate_gas.py
+++ b/util/test/test_DFRewards_allocate_gas.py
@@ -49,20 +49,20 @@ def test_insufficient_gas_reverts():
 
 
 @enforce_types
-def _batch_allocate(number: int) -> str:
+def _batch_allocate(n_accounts: int) -> str:
     account0 = brownie.network.accounts[0]
-    addresses, rewards, token_addr, df_rewards = _prep_batch_allocate(number)
+    addresses, rewards, token_addr, df_rewards = _prep_batch_allocate(n_accounts)
     tx = df_rewards.allocate(addresses, rewards, token_addr, {"from": account0})
     return tx
 
 
 @enforce_types
-def _prep_batch_allocate(number: int) -> Any:
+def _prep_batch_allocate(n_accounts: int) -> Any:
     account0 = brownie.network.accounts[0]
     OCEAN = oceanutil.OCEANtoken()
     df_rewards = B.DFRewards.deploy({"from": account0})
-    addresses = get_random_addresses(number)
-    rewards = [1 for i in range(number)]
+    addresses = get_random_addresses(n_accounts)
+    rewards = [1 for account_i in range(n_accounts)]
     OCEAN.approve(df_rewards, sum(rewards), {"from": account0})
     return addresses, rewards, OCEAN.address, df_rewards
 

--- a/util/test/test_DFRewards_allocate_gas.py
+++ b/util/test/test_DFRewards_allocate_gas.py
@@ -45,7 +45,8 @@ def test_insufficient_gas_reverts():
         df_rewards.allocate(
             addresses, rewards, token_addr, {"from": account0, "gas_limit": 100000}
         )
-    assert str(e_info.value) == "base fee exceeds gas limit"
+
+    assert "out of gas" in str(e_info.value)
 
 
 @enforce_types
@@ -58,6 +59,18 @@ def _batch_allocate(n_accounts: int) -> str:
 
 @enforce_types
 def _prep_batch_allocate(n_accounts: int) -> Any:
+    """
+    @description
+      Create 'n_accounts' random accounts, give each an OCEAN allowance.
+      To help testing of df_rewards
+
+    @return
+      addresses -- list[address_str]
+      rewards -- [1, 1, ..., n_accounts-1] -- reward in OCEAN per account
+      OCEAN_address - str
+      df_rewards -- DFRewards contract, controlled by account0.
+        Account0 approves it to spend sum(rewards)
+    """
     account0 = brownie.network.accounts[0]
     OCEAN = oceanutil.OCEANtoken()
     df_rewards = B.DFRewards.deploy({"from": account0})

--- a/util/test/test_DFRewards_allocate_gas.py
+++ b/util/test/test_DFRewards_allocate_gas.py
@@ -46,7 +46,8 @@ def test_insufficient_gas_reverts():
             addresses, rewards, token_addr, {"from": account0, "gas_limit": 100000}
         )
 
-    assert "out of gas" in str(e_info.value)
+    error_str = str(e_info.value)
+    assert "out of gas" in error_str or "intrinsic gas too low" in error_str
 
 
 @enforce_types

